src/ffmpeg.imageio/ffmpeginput.cpp

```diff
@@ -206,29 +213,31 @@ FFmpegInput::open (const std::string &name, ImageSpec &spec)
         !strcmp (m_codec_context->codec->name, "dvvideo")) {
         m_offset_time = false;
     }
+
+   m_codec_cap_delay=m_codec_context->codec->capabilities&AV_CODEC_CAP_DELAY ;
```

Comment:
We can swich AV_CODEC_CAP_DELAY to CODEC_CAP_DELAY.
https://ffmpeg.org/doxygen/2.8/libavcodec_2avcodec_8h_source.html#l01064




2015-12-10 19:08 GMT+01:00 Larry Gritz <notifications@github.com>:

> In src/ffmpeg.imageio/ffmpeginput.cpp
> <https://github.com/OpenImageIO/oiio/pull/1288#discussion_r47261475>:
>
> > @@ -206,29 +213,31 @@ FFmpegInput::open (const std::string &name, ImageSpec &spec)
> >          !strcmp (m_codec_context->codec->name, "dvvideo")) {
> >          m_offset_time = false;
> >      }
> > +
> > +   m_codec_cap_delay=m_codec_context->codec->capabilities&AV_CODEC_CAP_DELAY ;
>
> In the version of ffmpeg on one of my systems, AV_CODEC_CAP_DELAY is not
> found.
>
> But there is a CODEC_CAP_DELAY. Are they the same, and I'm using a
> slightly older version, after which the AV_ was prepended?
>
> do you think it's sufficient if I just add something like:
>
> #ifndef AV_CODEC_CAP_DELAY
> #define AV_CODEC_CAP_DELAY CODEC_CAP_DELAY
> #endif
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenImageIO/oiio/pull/1288/files#r47261475>.
>